cmd/webhook/main.go

```diff
@@ -213,6 +215,22 @@ func main() {
 		SecretName:  secretName,
 	})
 
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("/", handler)
+	mux.HandleFunc("/health", healthHandler)
+	mux.HandleFunc("/readiness", readinessHandler)
+
+	// use PORT environment variable, or default to 8080
```

Comment:
The code below this line is self-explanatory.  I don't think the comment is necessary.